### PR TITLE
Update building-a-native-ios-app.md

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

File: content/yaml-quick-start/building-a-native-ios-app.md [hunk 1]
```diff
@@ -46,6 +46,37 @@ If you have Xcode Debugging Symbols enabled, the dSYM file will be generated in
 ```yaml
 $HOME/Library/Developer/Xcode/DerivedData/**/Build/**/*.dSYM
 ```
+Configuring the yaml file like the following creates a zip file and **.app** inside it:
+
+```yaml
+workflows:
+    react-native-ios:
```
Comment (nevercode-kevin) [inline content/yaml-quick-start/building-a-native-ios-app.md:53]:
```suggestion
    simulator-native-ios:
```

File: content/yaml-quick-start/building-a-native-ios-app.md [hunk 2]
```diff
@@ -46,6 +46,37 @@ If you have Xcode Debugging Symbols enabled, the dSYM file will be generated in
...
+            vars:
+                XCODE_WORKSPACE: "your_workspace_name.xcworkspace" # <-- Put the name of your Xcode workspace here
+                XCODE_SCHEME: "your_workspace_name" # <-- Put the name of your Xcode scheme here
+            node: v16.11.1
+            xcode: 13.0
+            cocoapods: default
+        scripts:
+            - name: Install npm dependencies
+              script: |
+                npm install
```
Comment (nevercode-kevin) [inline content/yaml-quick-start/building-a-native-ios-app.md:67]: Not required in a native ios project.

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

**Comment by icarusdust** [comment]
Done!

File: content/yaml-quick-start/building-a-native-ios-app.md [hunk 3]
```diff
@@ -46,6 +46,34 @@ If you have Xcode Debugging Symbols enabled, the dSYM file will be generated in
...
+workflows:
+    simulator-native-ios:
+        name: iOS simulator build
+        max_build_duration: 120
+        instance_type: mac_mini
+        environment:
+            vars:
+                XCODE_WORKSPACE: "your_workspace_name.xcworkspace" # <-- Put the name of your Xcode workspace here
+                XCODE_SCHEME: "your_workspace_name" # <-- Put the name of your Xcode scheme here
+            node: v16.11.1
```
Comment (nevercode-kevin) [inline content/yaml-quick-start/building-a-native-ios-app.md:61]: Not required for native iOS build.

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

File: content/yaml-quick-start/building-a-native-ios-app.md [hunk 4]
```diff
@@ -55,10 +55,6 @@ workflows:
         max_build_duration: 120
         instance_type: mac_mini
         environment:
-            vars:
-                XCODE_WORKSPACE: "your_workspace_name.xcworkspace" # <-- Put the name of your Xcode workspace here
-                XCODE_SCHEME: "your_workspace_name" # <-- Put the name of your Xcode scheme here
-            node: v16.11.1
```
Comment (nevercode-kevin) [inline content/yaml-quick-start/building-a-native-ios-app.md]: Only need to delete `node: v16.11.1`

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**